components/admin_console/billing/company_info_edit.tsx

```diff
@@ -28,10 +28,6 @@ const CompanyInfoEdit: React.FC<Props> = () => {
     const dispatch = useDispatch();
     const companyInfo = useSelector((state: GlobalState) => state.entities.cloud.customer);
 
-    if (!companyInfo) {
```

Comment:
This is probably the biggest change we'll need to make for stuff like this. We can't do early returns before hooks are used since they can't be called conditionally as per the rules of hooks.